pandas/tests/arithmetic/test_numeric.py

```diff
@@ -833,6 +833,15 @@ def test_ufunc_coercions(self, holder):
         exp = tm.box_expected(exp, box)
         tm.assert_equal(result, exp)
 
+    @pytest.mark.parametrize('holder', [pd.Int64Index, pd.UInt64Index,
+                                        pd.Float64Index, pd.Series])
+    def test_ufunc_reduce(self, holder):
+        idx = holder([1, 2, 3, 4, 5], name='x')
+
+        result = np.add.reduce(idx)
```

Comment:
use paramterize over some more ops here

Reply:
We would only be testing that those work on the underlying numpy array (which they do :)), so I don't think it is that important

Reply:
yes exactly